modules/client/templates/scripts/forseti-client/run_forseti.sh.tpl

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
I don't recall the client box needing a `run_forseti.sh`.  Has it always needed it and I just not remember?

Reply:
Correct, client box doesn't need `run_forseti.sh` and [so it is not there](https://github.com/forseti-security/terraform-google-forseti/tree/migrate-run-forseti-script/modules/client/templates/scripts/forseti-client).